test/integration/targets/win_chocolatey_facts/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: ensure Chocolatey is installed
+  win_chocolatey:
+    name: chocolatey
+    state: present
+
+- block:
+  - name: run tests
```

Comment:
No need for the main/test split. You can just include all the tests in `tasks/main.yml` as you aren't cleaning up anything.